git-sync/main.go

```diff
@@ -95,16 +98,29 @@ func main() {
 		}
 	}
 
+	initialSync := true
+	failCount := 0
 	for {
 		if err := syncRepo(*flRepo, *flDest, *flBranch, *flRev, *flDepth); err != nil {
-			log.Fatalf("error syncing repo: %v", err)
+			if initialSync || *flMaxSyncFailures == 0 || (*flMaxSyncFailures > 0 && failCount >= *flMaxSyncFailures) {
```

Comment:
Is `-1` supposed to mean 'forever' ?

Isn't `*flMaxSyncFailures == 0 || (*flMaxSyncFailures > 0 && failCount >= *flMaxSyncFailures)` just: `failCount >= *flMaxSyncFailures` ?